src/client/sandbox/code-instrumentation/methods.js

```diff
@@ -1,12 +1,46 @@
 import SandboxBase from '../base';
 import { isNullOrUndefined, inaccessibleTypeToStr, isWindow, isDocument } from '../../utils/types';
 import { DOCUMENT_WRITE_BEGIN_PARAM, DOCUMENT_WRITE_END_PARAM, CALL_METHOD_METH_NAME } from '../../../processing/js';
+import { FOCUS_PSEUDO_CLASS_ATTR } from '../../../const';
+import { isDomElement } from '../../utils/dom';
```

Comment:
Why we have `isDocument`, `isWindow`, etc. in `utils/types` while `isDomElement` is in `utils/dom`?
\cc @churkin